code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -260,7 +260,6 @@
 	new /obj/item/clothing/accessory/holster(src)
 	new /obj/item/clothing/accessory/blue(src)
 	new /obj/item/clothing/shoes/jackboots/jacksandals(src)
-	new /obj/item/clothing/under/rank/centcom/blueshield(src)

```

Comment:
This is removing the blueshield CC outfit

Reply:
Edit for this point (GH doesnt let you edit review comments for obvious reasons)
Same for the what I said with the NT rep. Because its blueshield specific, its fine.